Tests/FluentKitTests/AsyncTests/AsyncFilterQueryTests.swift

```diff
@@ -15,31 +15,64 @@ final class AsyncFilterQueryTests: XCTestCase {
         let db = DummyDatabaseForTestSQLSerializer()
```

Comment:
PR itself LGTM, but you should move these `let db = DummyDatabaseForTestSQLSerializer()` and `db.reset()` calls to `setup()` and `tearDown()` functions.

Reply:
That is nowhere remotely near important enough to be worth blocking a bugfix. If it was a new problem that'd be one thing, but the Fluent tests have been a mess and a half for a long time.

Reply:
Hmm i guess not blocking, but i'd like the change still 😝